server/functions.lua

```diff
@@ -408,6 +416,7 @@ exports('GetCoreVersion', GetCoreVersion)
 local function ExploitBan(playerId, origin)
     local name = GetPlayerName(playerId)
     local success, errorResult = pcall(
+        ---@diagnostic disable-next-line: param-type-mismatch
```

Comment:
What is even going on here? Is this code even valid? `pcall` is supposed to execute a function given to it.

Reply:
Absolutely no idea

Reply:
Just remove the `pcall` part, which will also let us remove this diagnostic comment. The `success` and `errorResult` come from the `insertBan` function directly.